src/core/thread/mle_router.hpp

```diff
@@ -494,6 +494,11 @@ class MleRouter : public Mle
         mDiscoveryRequestCallback.Set(aCallback, aContext);
     }
 
+    /**
+     * This method recalculates the MLE Advertisement Trickle timer interval based on SPEC-1167 algorithm.
```

Comment:
In the `main` branch now all the wording on "This method" and "This function" have been removed.

Also the description is incomplete, because the function also starts the Trickle timer in case it wasn't yet running.
That seems important enough to also have it stated in the function's name.
E.g. `RecalculateAdvertiseIntervalAndStartAdvertising()`